app/components/registration-photos/template.hbs

```diff
@@ -0,0 +1,33 @@
+<div class='collapse' id='collapseExample'>
```

Comment:
Maybe you can figure out better name for this `id` 😄 Then remember to changes in `app/register/template.hbs` file as well